src/cliente/scss/components/_producto.scss

```diff
@@ -0,0 +1,41 @@
+.c-producto {
+    margin-top: 15px;
```

Comment:
[Un componente no puede definir sus márgenes, de eso se encarga su contenedor](https://en.bem.info/methodology/css/#external-geometry-and-positioning), que vendría siendo `c-productos` que menciono en el comentario de la review.